docs/tutorial/units.rst

```diff
@@ -4,82 +4,133 @@
 Units
 *****
 
-This section of the guide talks about representing physical units in sunpy.
-sunpy makes use of the `astropy.units` for this task.
-
-Quantity objects
-================
-All functions in sunpy that accept or return numbers associated with physical quantities accept and return `~astropy.units.Quantity` objects.
+In this section of the guide, you will learn about representing physical units in sunpy.
```

Comment:
```suggestion
In this section of the tutorial you will learn about representing physical units in sunpy.
```
I'm also not sure whether this should be directly addressing the reader or not, but that's out of scope of this PR...

Reply:
Reading on I do like directly adressing the reader, perhaps worth converting the rest of our tutorial to that style?

Reply:
I'm in favor of that. Maybe Keep that style here in this PR and then we can create an issue to copy-edit the rest of the tutorial to be in that same style?